source/javascripts/filter-list.js

```diff
@@ -0,0 +1,58 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function FilterList ($module) {
+    this.$module = $module
+  }
+
+  FilterList.prototype.init = function () {
+    // find the input element
+    this.$module.input = this.$module.nodeName === 'INPUT' ? this.$module : this.$module.querySelector('input')
+    if(!this.$module.input) {
```

Comment:
Nitpick: `standardx` would add a space between `if(` here. E.g. `if (!this.$module.input) {`


Slightly off topic, but we could add `standardx` to this repo by running `yarn add standardx` and then adding this to the `package.json` - but I guess as it's not a part of the CI it would only be useful for devs.

```JSON
  "standardx": {
    "ignore": []
  },
  "eslintConfig": {
    "rules": {
      "no-var": 0
    }
  }
```

Reply:
Have changed. Yeah, standardx could be useful, let's think about it for a separate PR.